conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+#Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost/gobblin
```

Comment:
I think it's really a matter of if we want to allow user-defined working directory name. We have some plan to revisit the use of environment variables and we will definitely consider this suggestion.